### PR TITLE
[OKU-17]-Component of the Reviews Page 

### PR DESCRIPTION
**Pull request by twentymurial33**
## Summary of changes**
Reviews - Homepage

## Change Type
*Put an `x` in all the boxes that apply
- [] Bugfix (non-breaking change which fixes an issue)
- [X] Feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to change)

## Linear issue link
https://linear.app/oku-no-kumo/issue/OKU-17/reviews-homepage

## Before & after screenshots or video recordings (Before & After)

Screen Shot 2023-03-27 at 9 17 31 PM

### CODE REVIEW

File: project_app/pages/Main.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                  sx={{
+                    display: "flex",
+                    flexDirection: "column",
+                    width: "220px",
+                    height: "500px",
+                    border: "2px solid #000000",
+                    borderRadius: "10px",
+                    boxSizing: "border-box",
+                    left: "0px",
+                    top: "0px",
```
Comment (idiglove) [inline project_app/pages/Main.js:35]: can we have this config in a constants.ts file?

File: project_app/pages/Main.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  left: 34px;
+  top: 169px;
+  text-align: center;
+  font-family: "Rubik";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 14px;
+  color: #000000;
+`;
```
Comment (idiglove) [inline project_app/pages/Main.js:94]: have this in a separate styled.ts file

File: project_app/pages/Main.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  return (
+    <ThemeProvider theme={theme}>
+      <Wrapper>
+        <Container maxWidth="lg">
+          <Grid container spacing={8}>
+            {cards.map((card) => (
+              <Grid item key={card} xs={8} sm={6} md={3}>
+                <Card
+                  sx={{
+                    display: "flex",
```
Comment (idiglove) [inline project_app/pages/Main.js:27]: can we have the whole Card component in a separate component pls thanks

**Review by idiglove — CHANGES_REQUESTED** [review]

File: project_app/pages/Main.js (hunk 2 above)
Comment (EstopaceMA) [reply project_app/pages/Main.js:94]: +1 to make it cleaner and maintainable.

File: project_app/pages/Main.js (hunk 3 above)
Comment (twentymurial33) [reply project_app/pages/Main.js:27]: yes will do

Comment (twentymurial33) [reply project_app/pages/Main.js:27]: Split code and have a separate Card component

File: project_app/pages/Main.js (hunk 2 above)
Comment (twentymurial33) [reply project_app/pages/Main.js]: Added `styled.ts` file

File: project_app/components/Card.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Image from "next/image";
+import { Card, Typography } from "./styled";
+
+function CardGame() {
+  return (
+    <Card sx={{}}>
+      <CardMedia
```
Comment (idiglove) [inline project_app/components/Card.js:10]: Hmm now Im thinking about it, can we rename the file and component name to `ReviewCard` ?

File: project_app/components/styled.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
+import styled from "styled-components";
+
+export const Typography = styled.text`
+  width: 152px;
+  height: 86px;
+  left: 34px;
```
Comment (idiglove) [inline project_app/components/styled.tsx:6]: what is a styled.text? havent seen that one

File: project_app/pages/Main.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
+import Grid from "@mui/material/Grid";
+import Container from "@mui/material/Container";
+import styled from "styled-components";
+import CardGame from "../components/Card";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+
+const cards = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16];
```
Comment (idiglove) [inline project_app/pages/Main.js:7]: should be Main.tsx

**Review by idiglove — CHANGES_REQUESTED** [review]
Nice one, i have some changes requested. I need to run this on my local as well to see if the design requirements are met

File: project_app/pages/Main.js (hunk 6 above)
Comment (twentymurial33) [reply project_app/pages/Main.js:7]: Updated to Main.tsx @idiglove

File: project_app/components/Card.js (hunk 4 above)
Comment (twentymurial33) [reply project_app/components/Card.js:10]: Component renamed to `ReviewCard` @idiglove

File: project_app/pages/Reviews.tsx [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import CardBox from "../components/CardBox";
+import styled from "styled-components";
+
+
+
```
Comment (idiglove) [inline project_app/pages/Reviews.tsx:5]: remove unnecessary whitespaces

**Review by idiglove — CHANGES_REQUESTED** [review]
Nice one. But the card should be a bit yellowish, not white

File: project_app/components/ReviewCard.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
+import Image from "next/image";
+import { Card } from "./styled";
+
+function ReviewCard() {
+  return (
+    <Card>
+      <Image src="/Card.png" alt="dont know" width={220} height={330} />
```
[the remark this answers is not in the document]
Comment (twentymurial33) [reply project_app/components/ReviewCard.js]: Added the todo comment

File: project_app/pages/Reviews.tsx [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+export default function Main() {
+  return (
+    <div> <Container>
+          <CardBox/>
+        </Container>  </div>
+
+   
+    
+      
+  );
```
[the remark this answers is not in the document]
Comment (twentymurial33) [reply project_app/pages/Reviews.tsx]: updated

File: project_app/components/CardComponent.js [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+        <StyledText>
+          "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua...."
+        </StyledText>
+      </CardContent>
+      <CardActions>
+        <Image src="/Dice.png" width={35} height={35} alt="image" />
+        <div style={{ marginLeft: "100px" }}>
+          <Image src="/chat.png" width={12} height={12} alt="image" />
+          <Image src="/love.png" width={12} height={12} alt="image" />
```
[the remark this answers is not in the document]
Comment (twentymurial33) [reply project_app/components/CardComponent.js:23]: Ok sounds good

File: project_app/components/CardBox.js [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+        <MazeContainer>
+          <Image
+            src="/Maze.png"
+            fill="absolute"
+            objectFit="cover"
+            alt="picture of the background"
+          />
+        </MazeContainer>
+
+        {/* <div
```
Comment (idiglove) [inline project_app/components/CardBox.js:25]: remove comments

File: project_app/components/CardBox.js [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+        </div> */}
+        <LeftWaterContainer>
+          <Image
+            src="/left_review.png"
+            width={103.74}
+            height={343.85}
+            alt="image"
+          />
+        </LeftWaterContainer>
+        <Grid
```
Comment (idiglove) [inline project_app/components/CardBox.js:38]: the water looks weird, but we can fix that later

**Review by idiglove — CHANGES_REQUESTED** [review]
some changes

File: project_app/components/CardBox.js (hunk 12 above)
Comment (twentymurial33) [reply project_app/components/CardBox.js:38]: Copy that

**Review by idiglove — APPROVED** [review]
lgtm

**Pull request merged**